utils/build/update_canary_version.js

```diff
@@ -19,10 +19,20 @@ const fs = require('fs');
 const path = require('path');
 const { execSync } = require('child_process');
 
-const timestamp = execSync('git show -s --format=%ct HEAD', {
-  stdio: ['ignore', 'pipe', 'ignore']
-}).toString('utf8').trim() + '000';
 const packageJSON = require('../../package.json');
-packageJSON.version = packageJSON.version + '-' + timestamp;
+if (process.argv[2] === '--today-date') {
+  const date = new Date();
+  const month = ['jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec'][date.getMonth()];
+  const day = date.getDate();
+  const year = date.getFullYear();
+  packageJSON.version = `${packageJSON.version}-alpha-${month}-${day}-${year}`;
```

Comment:
Just curious, why do we need "alpha"?

Reply:
just to ensure that all there versions go before all others (there's alphabetical version sorting)